test/test_run_analysis.rb

```diff
@@ -9,21 +9,37 @@
 class TestResStockMeasuresOSW < MiniTest::Test
   def before_setup
     cli_path = OpenStudio.getOpenStudioCLI
-    @command = "\"#{cli_path}\" workflow/run_analysis.rb -y "
+    @command = "\"#{cli_path}\" workflow/run_analysis.rb"
+    @workflowdir = File.join(File.dirname(__FILE__), '..', 'workflow')
+  end
+
+  def test_version
+    @command += ' -v'
+
+    system(@command)
   end
```

Comment:
@shorowit I was trying to include one. But found that even when the call succeeded, it would return `false`. Maybe there's a way to get stdout from the `system` call?

Reply:
Or maybe it needs to be:
```
stdout = `#{@command}`
```